astropy/io/fits/tests/test_header.py

```diff
@@ -562,6 +562,16 @@ def test_continue_card_with_equals_in_value(self):
                 '* 5.87359e-12 * MWAvg(Av=0.12)')
         assert c.comment == 'pysyn expression'
 
+    def test_final_continue_card_lacks_ampersand(self):
+        """
+        Regression test for https://github.com/astropy/astropy/issues/3282
+        """
+
+        h = fits.Header()
+        h['SVALUE'] = 'A' * 69
+        assert repr(h).splitlines()[-1] == _pad("CONTINUE  'AA'")
+
+
```

Comment:
Super minor nitpick, but just remove one blank line here.  There should only be one blank line between each test method.

Otherwise, this test looks great.  I would just include the additional test case I suggested in [my other cmment](https://github.com/astropy/astropy/pull/3328/files#r23389133)